zerver/webhooks/opencollective/doc.md

```diff
@@ -1,19 +1,25 @@
-This integration currently supports getting notifications to a channel of your Zulip instance,
-when a new member signs-up on an **Open Collective** page.
+# Zulip Open Collective integration
+
+Get Zulip notifications for new sign-ups on your **Open Collective** page.
+
+{start_tabs}
 
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. Go to [Open Collective Website](https://opencollective.com/), find
-your desired collective page, then go to *Settings* -> *Webhooks*, paste the
-bot URL and choose *Activity* -> *New Member*.
+your desired collective page, then go to **Settings** > **Webhooks**, paste the
+generated URL, and choose **Activity** > **New Member**.
```

Comment:
The current instructions make it seem like you can set up this notification for any collective on the site, but really you can only create it for a collective you're an admin for. Also we should  separate out the navigation and configuration  steps.

```
1. In Open Collective, navigate to one of the collectives under
   **My Collectives**. Go to **Settings** > **Webhooks**.

1. Set **URL** to the URL generated above, and choose **Activity** >
   **New Member**.
```

Reply:
Seems to not have been updated for the comment above.